am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/helpers/DatabaseHelperRepository.java

```diff
@@ -59,6 +59,10 @@
         + "where resource_id = :resourceId")
     int countExplicitPermissions(String resourceId);
 
+    @SqlQuery("select relationship from access_management "
+        + "where resource_id = :resourceId")
+    String getNullRelationShipForAnnotation(String resourceId);
+
     @SqlQuery("select * from access_management "
         + "where resource_id = :resourceId")
```

Comment:
there is a remote possibility that resource id may repeat across different services/resource types

Therefore, should be handled appropriately

Reply:
right may be we can use existing method findExplicitPermissions method. Will use that